site/src/components/Mainabout.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
```

Comment:
The file should be under `sections`.
Create a folder named `About` and add then create an `index.js` file under it and the `about.style.js` file for CSS

Reply:
got it :+1: